src/utils/sorting-functions.ts

```diff
@@ -0,0 +1,123 @@
+import { ToDoItem, Priority } from '../models/todoitem-model';
+
+export function sortByTitle(items: ToDoItem[], order: number): ToDoItem[] {
+  const orderedItems = items;
+  if (order === 1) {
```

Comment:
The content of if and else are almost the same. Please do not duplicate the code, and create a helper function instead.